modules/fundamental/src/license/endpoints/mod.rs

```diff
@@ -1,93 +1,72 @@
+use crate::license::service::license_export::LicenseExporter;
 use crate::{
     Error,
     license::{
         endpoints::spdx::{get_spdx_license, list_spdx_licenses},
-        model::LicenseSummary,
+        get_sanitize_filename,
         service::LicenseService,
     },
 };
 use actix_web::{HttpResponse, Responder, get, web};
 use std::str::FromStr;
-use trustify_common::{
-    db::{Database, query::Query},
-    id::IdError,
-    model::{Paginated, PaginatedResults},
-};
-use uuid::Uuid;
+use trustify_common::{db::Database, id::Id};
 
 pub mod spdx;
 
-pub fn configure(config: &mut utoipa_actix_web::service_config::ServiceConfig, db: Database) {
-    let license_service = LicenseService::new(db);
+pub fn configure(config: &mut utoipa_actix_web::service_config::ServiceConfig) {
+    let license_service = LicenseService::new();
 
     config
         .app_data(web::Data::new(license_service))
         .service(list_spdx_licenses)
         .service(get_spdx_license)
-        .service(list_licenses)
-        .service(get_license)
-        .service(get_license_purls);
+        .service(get_license_export);
 }
 
 #[utoipa::path(
-    tag = "license",
-    operation_id = "listLicenses",
+    tag = "sbom",
+    operation_id = "getLicenseExport",
     params(
-        Query,
-        Paginated,
+        ("id" = String, Path, description = "Digest/hash of the document, prefixed by hash type, such as 'sha256:<hash>' or 'urn:uuid:<uuid>'"),
     ),
     responses(
-        (status = 200, description = "Matching licenses", body = PaginatedResults<LicenseSummary>),
+    (status = 200, description = "license gzip file", body = Vec<u8>),
+    (status = 404, description = "The document could not be found"),
     ),
 )]
-#[get("/v2/license")]
-/// List licenses
-pub async fn list_licenses(
-    state: web::Data<LicenseService>,
-    web::Query(search): web::Query<Query>,
-    web::Query(paginated): web::Query<Paginated>,
+#[get("/v2/sbom/{id}/license-export")]
```

Comment:
If this is an `sbom` API, then it should also be in the `sbom` module. Not in the `license` module.